features/citizens/citizens.feature

```diff
@@ -298,4 +298,24 @@ Feature: Citizen journey
     Then I should be on a page showing "What is your relationship to"
     Then I choose "They're a child relative"
     Then I click "Save and continue"
-    Then I should be on a page showing "dependants income"
+    Then I should be on a page showing "receive any income?"
+
+  @javascript
+  Scenario: I have an adult dependant more than 18 years old
```

Comment:
Rather than creating a new scenario, I suggest carrying on with the previous one.

Reply:
Carrying onto creating a new dependant that's more than 18 years old, will require the add assets page / feature which hasn't been implemented yet ( add income page - > add assets page - > any other dependants page )